mhr_api/src/mhr_api/models/db2/cmpserno.py

```diff
@@ -12,12 +12,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """This module holds data for legacy DB2 compressed serial number information."""
+import re
```

Comment:
Is this a typo? haha short module name!

Reply:
lol, I guess it is a short name module `key = re.sub('[^0-9A-Z]+', '', key)`